bin/cli.py

```diff
@@ -1,32 +1,41 @@
 from pathlib import Path
 import torch
 from torch import optim
-import torch.nn as nn
-import torch.nn.functional as F
-from torch.utils.data import DataLoader
-from torchvision import datasets, transforms
-
+# import torch.nn as nn

```

Comment:
you can remove these comments if they aren't being used :) Generally you never want to check in commented out code as it's unclear for someone else if that code is needed